### PR TITLE
Try to have travis use internal kokkos

### PR DESCRIPTION
**Pull request by jgfouca**
With this PR, we will have baseline testing against master in travis. We will also have all testing, Jenkins, travis, etc going through the same scripts.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ before_script:
 - gfortran --version
 
 script:
+- git config submodule.externals/kokkos.url https://github.com/kokkos/kokkos.git
```
Comment (bartgol) [inline .travis.yml]: Interesting. Was ssh not working?

Comment (jgfouca) [reply .travis.yml]: > Was ssh not working?

Unfortunately, it does not. Ssh access to github requires ssh keys to be set up, even for public repos.

**Comment by bartgol** [comment]
Uh, I missed the last 4 commits. Is the plan to then import your scream-scripts-mirror into scream?

File: .travis.yml (hunk 1 above)
Comment (bartgol) [reply .travis.yml]: Uh, true.

**Comment by jgfouca** [comment]
> Uh, I missed the last 4 commits. Is the plan to then import your scream-scripts-mirror into scream?

I plan on keeping scream-scripts-mirror in-sync with the few things I need from scream-docs. It's a hacky solution, but I really don't want to dump a bunch of python stuff into the main scream repo.

**Comment by jgfouca** [comment]
I forgot to merge this!

**Pull request merged**